test/object-store/sync/flx_sync.cpp

```diff
@@ -2329,8 +2317,11 @@ TEST_CASE("flx: interrupted bootstrap restarts/recovers on reconnect", "[sync][f
         realm->sync_session()->shutdown_and_wait();
     }
 
-    // Verify that the file was fully closed
-    REQUIRE(DB::call_with_lock(interrupted_realm_config.path, [](auto&) {}));
+    {
+        // Verify that the file was fully closed
+        auto empty = [](auto&) {};
+        REQUIRE(DB::call_with_lock(interrupted_realm_config.path, empty));
```

Comment:
Why did this change? How is this call different than what was there before?

Reply:
It doesn't really change anything
VS Code was reporting an error for this line: `a lambda is not allowed in an unevaluated expressionC/C++(1777)`